ocaml/piaf/dream-compat.nix

```diff
@@ -0,0 +1,73 @@
+{ stdenv
```

Comment:
piaf upstream is unvendored

Reply:
It's not compatible with dream at least 🤔

Reply:
I suggest you remove this from this PR and add it in a separate PR. And make sure to use current Piaf HEAD as you won't need the unvendor patch anymore.